backend/canvas.js

```diff
@@ -1,11 +1,10 @@
 import { MongoClient } from "mongodb";
-// require("dotenv").config();
-// using es6
-import "dotenv/config";
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
please stop modifying this. `import "dotenv/config` alone is the recomended usage for es6. You do not need to do dotenv.config()

Reply:
Okay I didn't know about that.